.gitignore

```diff
@@ -1,4 +1,6 @@
 /node_modules
 npm-debug.log
 yarn-error.log
+yarn.lock
+package-lock.json
```

Comment:
Why were the lock files added to the git ignore file?
They're used to ensure people get the same version of the packages when they install the bridge bot.

Reply:
I suppose I thought that the lock files weren't necessary for installation, seen as they are autogenerated anyway and the package versions are shown in the package.json file. Is that not the case?
![Versions](https://user-images.githubusercontent.com/65962094/111924868-da6bae80-8a7c-11eb-9440-46e6de869012.png)

Reply:
The lock files aren't necessary for setting up the app, however they help ensure that everyone who uses the app get the exact same versions of every package. An example for the `chalk` package is that its setup using `^` which would mean that `4.1.0` is a valid version for the project, but so is `4.1.1`, `4.1.2`, `4.1.3`, etc.
If something breaks in a newer version of a package which causes the app to crash or fail in some way, new people installing the packages without the lock file present will get the new version automatically, causing them to be unable to use the app, however, with the lock files in place they're ensured to get a version of the packages that work with the app, even if the version of the package is a few versions behind the latest version of whatever packages are installed.

Reply:
You can find a better explanation on how the lock files work here:

https://nodejs.dev/learn/the-package-lock-json-file